app/src/main/res/layout/app_language_activity.xml

```diff
@@ -1,5 +1,41 @@
 <?xml version="1.0" encoding="utf-8"?>
-<FrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
-  android:id="@+id/app_language_fragment_container"
-  android:layout_width="match_parent"
-  android:layout_height="match_parent" />
+<layout xmlns:android="http://schemas.android.com/apk/res/android"
+  xmlns:app="http://schemas.android.com/apk/res-auto">
+
+  <androidx.constraintlayout.widget.ConstraintLayout
+    android:layout_width="match_parent"
+    android:layout_height="match_parent">
+
+    <com.google.android.material.appbar.AppBarLayout
+      android:id="@+id/reading_list_app_bar_layout"
+      android:layout_width="0dp"
+      android:layout_height="wrap_content"
+      android:theme="@style/ThemeOverlay.AppCompat.Dark.ActionBar"
+      app:layout_constraintEnd_toEndOf="parent"
+      app:layout_constraintStart_toStartOf="parent"
+      app:layout_constraintTop_toTopOf="parent">
+
+      <androidx.appcompat.widget.Toolbar
+        android:id="@+id/app_language_toolbar"
+        android:layout_width="match_parent"
+        android:layout_height="?attr/actionBarSize"
+        android:background="?attr/colorPrimary"
+        android:fontFamily="sans-serif"
+        android:minHeight="?attr/actionBarSize"
+        app:navigationContentDescription="@string/navigate_up"
+        app:navigationIcon="?attr/homeAsUpIndicator"
+        app:title="@string/app_language_activity_title"
+        app:titleTextAppearance="@style/ToolbarTextAppearance"
+        app:titleTextColor="@color/white" />
+    </com.google.android.material.appbar.AppBarLayout>
+
+    <FrameLayout
+      android:id="@+id/app_language_fragment_container"
+      android:layout_width="0dp"
+      android:layout_height="0dp"
+      app:layout_constraintBottom_toBottomOf="parent"
+      app:layout_constraintEnd_toEndOf="parent"
+      app:layout_constraintStart_toStartOf="parent"
+      app:layout_constraintTop_toBottomOf="@id/reading_list_app_bar_layout" />
```

Comment:
try to add a closing tag for frame layout like </FrameLayout> instead of /> and then inside the frame layout below app_language_fragment_container you can add a shadow view.

Reply:
> try to add a closing tag for frame layout like instead of /> and then inside the frame layout below app_language_fragment_container you can add a shadow view.

I have added the app_language_toolbar_shadow_view in activity.xml , the way you had suggested but the view is still the same since shadow below the toolbar is still invisible.

|<img src="https://user-images.githubusercontent.com/78796264/155363531-da6c7f10-fceb-4d7e-a76a-d1ea1664c4e4.jpeg" width="350" />